pkg/cmd/p2p/env/sync/env_sync.go

```diff
@@ -73,7 +74,7 @@ func run(opts *EnvCreateOpts, cfg *config.Config) error {
 		return err
 	}
 
-	spinnerHandler := opts.Streams.Spinner("Configuring environments...")
+	spinnerHandler := opts.Streams.Wizard("Configuring environments", "Configured environments")
```

Comment:
maybe worth adding platform environment and not environment? Confusion with tenants environments

Reply:
Added